content/06.know-your-problem.md

```diff
@@ -18,8 +18,7 @@ All of these designs have different assumptions and caveats, which cannot be ign
 Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated.
 Samples originating from the same study site, oversampling of ethnic groups or zip codes, and sample processing differences are all sources of variation that need to be accounted for.
 
-Systematic biases can lead to artifacts or "batch effects," which mean that instead of finding correlates with an outcome or grouping of interest, the investigator may find correlates with variables that are not of interest and obtain misleading results [@doi:10.
-1038/nrg2825].
+Systematic biases can lead to artifacts or "batch effects," which mean that instead of finding correlates with an outcome or grouping of interest, the investigator may find correlates with variables that are not of interest and obtain misleading results [@doi:10.1038/nrg2825].
```

Comment:
I think there's some grammar error here: "the investigator ... and obtain misleading results ..."

Not sure if this is better, but here would be my rephrasing of the entire paragraph:

"Systematic biases, which can be induced by confounding variables, for example, can lead to artifacts or so-called "batch effects." As a consequence, models may learn to rely on correlations that are irrelevant in the scientific context of the study and may result in misguided predictions and misleading conclusions [@doi:10.1038/nrg2825]."

Reply:
Definitely better. Have fixed.